parm/config/config.resources.emc.dyn

```diff
@@ -176,6 +176,13 @@ elif [ $step = "analdiag" ]; then
     if [[ "$npe_node_analdiag" -gt "$npe_node_max" ]]; then export npe_node_analdiag=$npe_node_max ; fi
     export memory_analdiag="48GB"
 
+elif [ $step = "wdqms" ]; then
+
+    export wtime_wdqms="00:30:00"
+    export npe_wdqms=1
+    export nth_wdqms=1
+    export npe_node_wdqms=1
```

Comment:
1. The ecf script has 10 minutes for the walltime but this has 30 minutes. Please correct one or the other.
2. Add `export memory_wdqms="48GB"`; otherwise the rocoto xml won't get the shared flag because it doesn't see a memory request.

Reply:
Thanks for making adjustments!